src/frog-base.tsx

```diff
@@ -186,7 +185,64 @@ export type RouteOptions<method extends string = string> = Pick<
   (method extends 'frame' | 'image'
     ? {
         fonts?: ImageOptions['fonts'] | (() => Promise<ImageOptions['fonts']>)
-      }
+      } & (method extends 'frame'
+        ? {
+            preview?: {
+              /**
+               * Path of the next frame.
+               *
+               * @example '/submit'
+               */
+              action?: FrameResponse['action']
+              /**
+               * HTTP response headers.
+               */
+              headers?: FrameResponse['headers']
+              /**
+               * An icon ID.
+               *
+               * @see https://warpcast.notion.site/Spec-Farcaster-Actions-84d5a85d479a43139ea883f6823d8caa
+               * @example `'log'`
+               */
+              icon?: Octicon
+              /**
+               * Title of the frame (added as `og:title`).
+               *
+               * @example 'Hello Frog'
+               */
+              title?: FrameResponse['title']
```

Comment:
not sure if we can inherit those docs from `FrameResponse` – without the comment it doesn't seem to pull up.